src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<blockquote>It also imposes to be consistent with the units of the data which is not always the case in the aircraft design formula, an example being the formula we used for the wing mass.</blockquote><p>It also imposes to be consistent with the units of the data<code>,</code> which is not always the case in the aircraft design formula <code>(</code>an example being the formula we used for the wing mass<code>)</code>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>